projects/pastanaga-angular/src/lib/avatar-pile/avatar-pile.component.ts

```diff
@@ -23,15 +23,15 @@ export class AvatarPileComponent {
     }
 
     @Input()
-    set customButton(value: boolean) {
+    set customButton(value: any) {
```

Comment:
it does accept a boolean, but then when using the single `customButton` syntax in the template, the compilation fails saying a boolean does not accept a string as value.
So whenever we use `coerceBooleanProperty`, we need to use `any` as setter type to prevent errors in the template when used as an attribute without value.